study-builder/studies/pancan/study-events.conf

```diff
@@ -669,15 +669,36 @@
         "type": "ACTIVITY_INSTANCE_CREATION",
         "activityCode": ${id.act.about_cancer},
         "createFromAnswer": true,
-        "sourceQuestionStableId": ${id.q.primary_cancer_list},
+        "sourceQuestionStableId": ${id.q.primary_cancer_list_self},
         "targetQuestionStableId": ${id.q.diagnosis_type}
       },
       # If there's already about-cancer instances, then don't create any more.
-      "preconditionExpr": """!user.studies["cmi-pancan"].forms["ABOUT_CANCER"].hasInstance()""",
+      "preconditionExpr": """!user.studies["cmi-pancan"].forms["ABOUT_CANCER"].hasInstance()
+      && (user.studies["cmi-pancan"].forms["PREQUAL"].questions["DESCRIBE"].answers.hasOption("DIAGNOSED"))""",
       "maxOccurrencesPerUser": 1,
       "dispatchToHousekeeping": false,
       "order": 1
     },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": ${id.act.release},
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": ${id.act.about_cancer},
+        "createFromAnswer": true,
+        "sourceQuestionStableId": ${id.q.primary_cancer_list_child},
+        "targetQuestionStableId": ${id.q.diagnosis_type}
+      },
+      # If there's already about-cancer instances, then don't create any more.
+      "preconditionExpr": """!user.studies["cmi-pancan"].forms["ABOUT_CANCER"].hasInstance()
+      && """ ${_pex.is_child_only},
```

Comment:
Wait a minute... why do we need another one of this and have it look at child? If they have a `release` activity it means they are doing the self flow (only self or self+child). And if they have a `release_minor` activity then they're going through the child flow and they have answered the child cancer list question.

I don't think we need another one of these?